src/components/pages/matchTrial/style.js

```diff
@@ -0,0 +1,158 @@
+import styled from 'styled-components';
+import { colors, breakpoint } from '../../../styles/globalStyles';
+
+export const Wrapper = styled.section`
+  background-color: white;
+  padding: 1.7rem 1rem 1rem 1rem;
+`;
+
+export const DetailWrapper = styled.div`
+  border-bottom: solid 1px;
+  padding: 1rem 0rem;
+  margin-bottom: 0.7rem;
+  display: 'flex';
+  flex-direction: 'column';
+`;
+export const DetailSection = styled.div`
+  display: flex;
+  justify-content: space-between;
+  line-height: 28px;
+  @media only screen and (max-width: ${breakpoint.tablet}) {
+    display: flex;
+    flex-direction: column;
+    & > button {
+      margin-top: 10px;
+      align-self: flex-end;
+    }
+  }
+`;
+
+export const HighLight = styled.span`
+  color: ${colors.primary};
+  text-decoration: underline;
+  font-size: 1rem;
+`;
+
+export const HighLightNumber = styled.span`
+  color: ${colors.cancel};
+  font-size: 2rem;
+  padding-left: 1rem;
+`;
+
+export const PrimarySpam = styled.span`
+  color: ${colors.primary};
+`;
+
+export const PrimaryParagraph = styled.p`
+  color: ${colors.primary};
+`;
+
+export const BoldParagraph = styled.p`
+  color: ${colors.black};
+  padding: 0.7rem 0rem 0rem;
+  font-weight: bold;
+`;
+
+export const HeadSection = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const ExportButton = styled.button`
+  border: none;
+  outline: none;
+  color: #fff;
+  background-color: #35348f;
+  text-align: center;
+  padding: 0.2rem 0.5rem;
+  border-radius: 6px;
+`;
+
+export const HeadPragraph = styled.p`
+  padding: 1rem 0rem;
+  font-weight: 900;
```

Comment:
I realise there is no variable defined for this in `globalStyles.js`, which we may want, just to make things clear.
In `Typography.js`, we don't import a font with weight of 900. Use **700** instead:

<img width="704" alt="Screenshot 2019-10-13 at 12 01 18" src="https://user-images.githubusercontent.com/28146666/66714674-7b970580-edb1-11e9-9c22-c80851b1da27.png">